pkgs/by-name/ca/canokey-usbip/package.nix

```
@@ -0,0 +1,42 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  python3,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "canokey-usbip";
+  version = "unstable-2024-03-11";
```

```suggestion
  version = "0-unstable-2024-03-11";
```

This would be better (If you are not using that git updater?)

---

Changed to prefix version with `0-` and add `unstableGitUpdater` to passthru